### PR TITLE
Added unit tests for CallVisualizer

### PR DESCRIPTION
**Pull request by EgorovEI**
MOB-1943

### CODE REVIEW

**Review by rasmustautsglia — APPROVED** [review]

File: GliaWidgetsTests/CallVisualizer/CallVisualizerTests.swift [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import XCTest
+import SalemoveSDK
+@testable import GliaWidgets
+
+final class CallVisualizerTests: XCTestCase {
+    func testScreenSharingStateChanged() {
+        enum Call { case updateState }
+        var calls: [Call] = []
```
Comment (yurii-glia) [inline GliaWidgetsTests/CallVisualizer/CallVisualizerTests.swift:8]: I'd like to propose to move enum definition and calls var on one level top.
```suggestion
final class CallVisualizerTests: XCTestCase {
    enum Call { case updateState }
    var calls: [Call] = []
    
    override func setUp() throws { 
        super.setUp()
        calls.removeAll()
     }
    
    func testScreenSharingStateChanged() {
```

**Review by yurii-glia — APPROVED** [review]

File: GliaWidgetsTests/CallVisualizer/CallVisualizerTests.swift (hunk 1 above)
Comment (EgorovEI) [reply GliaWidgetsTests/CallVisualizer/CallVisualizerTests.swift]: 3337cd283a4de40419b8ccd27f3e4d4371aeb22d

**Review by igorkravchenko — APPROVED** [review]
Looks good 👍

**Comment by EgorovEI** [comment]
!squash

**Pull request merged**